test/algorithms/perimeter/polygon_cases.hpp

```diff
@@ -0,0 +1,46 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+// Unit Test
+
+// Copyright (c) 2016 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fisikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef POLYGON_CASES_HPP
+#define POLYGON_CASES_HPP
+
+#include <string>
+
+static std::string Poly_data_geo_arr[] = {
```

Comment:
Should not start with a capital, and curly brace is normally placed on next line in our conventions.

Maybe POLYGON_CASES_HPP can be preceded by perimeter, for in case there is another file like this (might be probable)

The _arr suffix is not necessary in my opinion, I usually have things like:

static std::string case_recursive_boxes_28[2] =
{
    "MULTIPOLYGON(((5 1,5 0,4 0,4 2,5 3,5 1)),((4 2,3 2,3 3,4 3,4 2)))",
    "MULTIPOLYGON(((2 2,2 3,3 3,4 2,3 1,2 1,2 2)),((3 4,4 3,3 3,3 4)),((4 2,5 2,4 1,4 2)))"
};

The poly prefix (which I did not use) is actually quite useful.